MetadataExtractor/Formats/QuickTime/QuickTimeMetadataReader.cs

```diff
@@ -176,6 +178,22 @@ void Handler(AtomCallbackArgs a)
                         QuickTimeReader.ProcessAtoms(stream, MoovHandler, a.BytesLeft);
                         break;
                     }
+                    case "uuid":
+                    {
+                        var XMP = new byte[] { 0xbe, 0x7a, 0xcf, 0xcb, 0x97, 0xa9, 0x42, 0xe8, 0x9c, 0x71, 0x99, 0x94, 0x91, 0xe3, 0xaf, 0xac };
+                        if (a.BytesLeft >= XMP.Length)
+                        {
+                            var uuid = a.Reader.GetBytes(XMP.Length);
+                            if (XMP.RegionEquals(0, XMP.Length, uuid))
+                            {
+                                var xmpBytes = a.Reader.GetNullTerminatedBytes((int)a.BytesLeft);
+                                var xmpDirectory = new XmpReader().Extract(xmpBytes);
+                                xmpDirectory.Parent = directories.OfType<ExifIfd0Directory>().SingleOrDefault();
```

Comment:
This will throw if there is more than one `ExifIfd0Directory`, which can happen. We should be explicit about parents, though in this case it looks like there may not be a parent directory, and that this is a top level directory.